core/cmd/solana_transaction_commands.go

```diff
@@ -0,0 +1,76 @@
+package cmd
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strconv"
+
+	solanaGo "github.com/gagliardetto/solana-go"
+	"github.com/urfave/cli"
+	"go.uber.org/multierr"
+
+	"github.com/smartcontractkit/chainlink/core/store/models/solana"
+)
+
+// SolanaSendSol transfers sol from the node's account to a specified address.
+func (cli *Client) SolanaSendSol(c *cli.Context) (err error) {
```

Comment:
This is not tested yet. The Terra version uses an integration test. What are our options for Solana?
- [ ] add test

Reply:
typically i have just mocked the API response, but we could also set up a similar integration test with a locally run solana validator?

Reply:
did some initial testing and fixes: https://github.com/smartcontractkit/chainlink/pull/6231/commits/af3d35a777ab101598bc598c6aa72bdadf619be7

but still not in working condition - logs a tx hash and cli returns fine, but nothing on chain 🤔

Reply:
ok, verified this works properly! (i was trying to send SOL to an account that can't accept SOL)

future work item is for transaction simulation

Reply:
Can we have a test on it?

Reply:
tests have been added to the tx manager + transfer cli command!